packages/FormElement/src/components/Label/Label.js

```diff
@@ -2,17 +2,17 @@ import React from "react";
 import PropTypes from "prop-types";
 import useI18n from "@paprika/l10n/lib/useI18n";
 import tokens from "@paprika/tokens";
+import Popover from "@paprika/popover";
 import Help from "../Help/Help";
 import { FormElementContext } from "../../FormElement";
 import * as sc from "./Label.styles";
 
 const Label = props => {
-  const { help, helpA11yText, children, isDisabled: isDisabledProp, ...moreProps } = props;
+  const { help, tooltipAlignment, helpA11yText, children, isDisabled: isDisabledProp, ...moreProps } = props;
```

Comment:
Maybe exposing the `<Help>` component for consumers as a prop collector would be better, but I think this approach you have is much simpler, and probably a better choice for now.  

But I would recommend `helpAlign` for the prop name instead to more closely align with the names of the related props.